integration-tests/src/test/resources/features/action-webhook.feature

```diff
@@ -25,7 +25,7 @@ Feature: Webhook Action tests
     And the list of Processor instances of the Bridge "mybridge" is containing the Processor "myProcessor"
     And the Processor "myProcessor" of the Bridge "mybridge" is existing with status "ready" within 3 minutes
     And the Processor "myProcessor" of the Bridge "mybridge" has action of type "webhook_sink_0.1" and parameters:
-      | endpoint | https://webhook.site/${env.webhook.site.uuid} |
+      | endpoint | https://webhook.site/${webhook.site.token.first} |
```

Comment:
@sutaakar shouldn't we now change the naming also in `yaml` workflow files? there we still use `UUID`? Shouldn;t also there `TOKEN` be used?

Reply:
yes, the workflow files can be adjusted too

Reply:
@sutaakar should I do it?

Reply:
@jomarko feel free to do so

Reply:
@jomarko feel free to prepare a change in workflow files, I will adjust the secrets then

Reply:
@sutaakar FYI https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/1224